video/out/w32_common.c

```diff
@@ -73,7 +74,7 @@ struct w32_api {
 struct vo_w32_state {
     struct mp_log *log;
     struct vo *vo;
-    struct mp_vo_opts *opts;
```

Comment:
Keeping this would have been fine. It'd just be the value of opts_cache->opts. (That pointer never changes.)

Reply:
Done.